modules/ding_frontend/ding_frontend.module

```diff
@@ -268,3 +268,61 @@ function ding_frontend_search_api_views_query_alter(view &$view, SearchApiViewsQ
     }
   }
 }
+
+function ding_frontend_form_ting_search_sort_form_alter(&$form, &$form_state, $form_id) {
```

Comment:
These changes are related to the frontend for the search part of the project. which we call `ting`. Consenquently this should be moved to the `ding_ting_frontend` module.